app/js/actions/pastOrders.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { LOAD_PAST_ORDERS_SUCCESS, LOAD_PAST_ORDERS_FAILURE } from './actionTypes';
+
+const contextPath = window.location.href.split('/')[3];
+const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
+
+export const getPastOrdersSuccess = pastOrders => ({
+  type: LOAD_PAST_ORDERS_SUCCESS,
+  pastOrders,
+});
+
+export const getPastOrdersFailure = error => ({
+  type: LOAD_PAST_ORDERS_FAILURE,
+  error,
+});
+
+export const getPastOrders = (patientUuid, careSetting) => dispatch => axios.get(`${apiBaseUrl}/order?careSetting=${careSetting}&patient=${patientUuid}&status=inactive&t=drugorder&v=full`)
+  .then((response) => {
+    dispatch(getPastOrdersSuccess(response.data.results));
+  })
+  .catch((err) => {
+    dispatch(getPastOrdersSuccess(err));
```

Comment:
Isn't this meant to be `dispatch(getPastOrdersFailure(err));`?

Reply:
Oooh yes, Thank u